Doc/whatsnew/3.14.rst

```diff
@@ -348,12 +348,18 @@ dis
   This feature is also exposed via :option:`dis --show-positions`.
   (Contributed by Bénédikt Tran in :gh:`123165`.)
 
+* Add the :option:`dis --specialized` command-line option to
+  show specialized bytecode.
+  (Contributed by Bénédikt Tran in :gh:`127413`.)
+
+
 errno
 -----
 
 * Add :data:`errno.EHWPOISON` error code.
   (Contributed by James Roy in :gh:`126585`.)
 
+
```

Comment:
This line was added after I fixed the conflict. Since we put two blank lines between sections, I think it's ok to keep but I can revert it.

Reply:
(At some point, we'll need a copy-edit because the above sections are not splitted with double lines while the sections below are... But since most of the sections have two blank lines, it's easier to add them to places that lack them).